src/types/response.rs

```diff
@@ -28,7 +28,9 @@ impl Deserialize for TxLogResponse {
             #[cfg(not(feature = "time_as_str"))]
             tx___tx_time: tx_time
                 .parse::<DateTime<FixedOffset>>()
-                .map_err(|_| EdnError::Deserialize("Unable to deserialize `:crux.tx/tx-time`, verify if the transaction time you're sending is correct".to_string()))?,
+                .map_err(|_| 
```

Comment:
I am confused on why this changed, was it because I pushed it? :thinking: 
I mean, the project uses `cargo fmt` already.